src/lib/remark-plugins/rewrite-tutorial-links/index.ts

```diff
@@ -75,15 +78,20 @@ function handleRewriteTutorialsLink(node: Link | Definition) {
       const isTutorialPath = nodePath.includes('tutorials')
       const learnProductHub = new RegExp(`/${product}$`)
       const isProductHubPath = learnProductHub.test(nodePath)
+      const isDocsPath = nodePath.includes(PRODUCT_DOCS_PATHS[product])
 
       // if its an external link, isolate the pathname
-      if (isExternalLearnLink) {
+      if (isExternalLearnLink || isDocsPath) {
         const fullUrl = new URL(nodePath)
-        nodePath = fullUrl.pathname
+        // removing the origin from the href instead of only using
+        // 'pathname' so that anchor links are included
+        nodePath = fullUrl.href.replace(fullUrl.origin, '')
```

Comment:
This was a fix for anchor links....Previously when just using `pathname`, the anchor link would be dropped.